amida-tech/indaba-client
# INBA-134 fix badge styling

#### What's this PR do?
Make badge styling conformant with BEM and also fix the styling

#### Related JIRA tickets:
[INBA-134](https://jira.amida-tech.com/browse/INBA-134)

#### How should this be manually tested?
1. Load http://localhost:3000/project/101
2. Click on the Users tab, and the User Groups tab within that
3. Check that the badges render correctly

#### Any background context you want to provide?
#### Screenshots (if appropriate):
